### ISSUE TITLE
Fix MKV Thumbnail embedding

### ISSUE THREAD

**Issue by Taurus3ye** [blackjack4494/youtube-dlc#126]
If you try to embed Thumbnails in an .mkv container you get "ERROR: Only mp3 and m4a/mp4 are supported for thumbnail embedding for now."

Youtube-dl already has a checked pull request [youtube-dl pull: 22659](https://github.com/ytdl-org/youtube-dl/pull/22659)

**Comment by blackjack4494** [comment]
@Taurus3ye can you give some sample command you used? It used to work for me when I tested it.  
https://github.com/blackjack4494/youtube-dlc/pull/28

**Comment by Taurus3ye** [comment]
> 
> 
> @Taurus3ye can you give some sample command you used? It used to work for me when I tested it.
> #28

-f bestvideo+bestaudio -o "[Path]\videos\%(title)s.%(ext)s" --embed-thumbnail

**Comment by cpz3501** [comment]
Same error text here, really looking forward for this feature.

**Comment by cpz3501** [comment]
It seems that the error only occurs when downloading to a different hard drive than the one with the OS.

**Comment by blackjack4494** [comment]
> It seems that the error only occurs when downloading to a different hard drive than the one with the OS.
> 
> Edit: that's the error i got "OSError: [WinError 17] The system cannot move the file to a different disk drive."

Yes this got actually mentioned in a post. There is actually a simple solution for that (apparently). Also this issue should only occur on Windows.
